### ISSUE TITLE
Difference between run_sft.sh and run_psft.sh

### ISSUE THREAD

**Issue by zq0x7a0x71** [zwhong714/PSFT#7]
The only differences between run_sft.sh and run_psft.sh are the two hyperparameters, `train_prompt_mini_bsz` and `actor_rollout_ref.actor.optim.lr`. Is this sufficient to distinguish between the two distinct methods, SFT and PSFT?

**Comment by zwhong714** [comment]
One concern I have is the difference in batch sizes. For example, SFT uses a batch size of 256 with a learning rate of 2e-5, whereas PSFT uses a batch size of 32 with a learning rate of 1e-6 and updates $\pi_{\theta}^{\text{old}}$ eight times (to make the training curves comparable). To address this, we plan to report an additional SFT experiment with a batch size of 32 and a learning rate of 1e-6 for you. 

Could you kindly suggest any additional concerns? 

Many thanks!

**Comment by zq0x7a0x71** [comment]
@zwhong714 Thank you for your quick response. What I mean is that the only difference between these two startup scripts lies in their hyperparameters, and both invoke main_psft. I may have missed some key information, but it appears that both scripts are used to train models using the PSFT method.

**Comment by zwhong714** [comment]
Ok, the main difference may require referring to the training code. You can check the discussion in this issue: https://github.com/zwhong714/PSFT/issues/6

**Comment by zq0x7a0x71** [comment]
I see. Now, the RL framework is being used to simultaneously implement SFT and PSFT. Does this also mean that SFT performs an additional reference model rollout, causing unnecessary computational overhead?

**Comment by zwhong714** [comment]
Yes, the current version of the code does have this issue. We will solve this as soon as possible. 

Thank you for your careful code review!
